<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HPDA0161" name="com.qait.automation.stik.test.TC_028" tests="8" failures="5" timestamp="25 Feb 2014 10:42:49 GMT" time="32.518" errors="1">
  <testcase name="H_Verify_SeePlansAndPricing_Button" time="0.000" classname="com.qait.automation.stik.test.TC_028"/>
  <testcase name="E_Verify_Stay_Safe_Section" time="0.000" classname="com.qait.automation.stik.test.TC_028"/>
  <testcase name="setup" time="6.652" classname="com.qait.automation.stik.test.TC_028"/>
  <testcase name="G_Verify_Company_Page_Section" time="0.000" classname="com.qait.automation.stik.test.TC_028"/>
  <testcase name="B_Verify_Demo_SubSection_Under_Splash_Section" time="13.191" classname="com.qait.automation.stik.test.TC_028">
    <error type="java.lang.AssertionError" message="Rating in splash section not visible expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Rating in splash section not visible expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertTrue(Assert.java:37)
	at com.qait.automation.stik.actionfixtures.DemoPageFixture.verifySplashSection(DemoPageFixture.java:13)
	at com.qait.automation.stik.test.TC_028.B_Verify_Demo_SubSection_Under_Splash_Section(TC_028.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- B_Verify_Demo_SubSection_Under_Splash_Section -->
  <testcase name="F_Verify_Stik_Premium_Section" time="0.000" classname="com.qait.automation.stik.test.TC_028"/>
  <testcase name="A_launch_URL_Navgiate_to_Stick_HomePage" time="12.674" classname="com.qait.automation.stik.test.TC_028"/>
  <testcase name="C_Verify_Get_Reviews_Section" time="0.001" classname="com.qait.automation.stik.test.TC_028"/>
</testsuite> <!-- com.qait.automation.stik.test.TC_028 -->
